src/Effect/Logger.hs

```diff
@@ -79,7 +79,7 @@ logError :: Has Logger sig m => Doc AnsiStyle -> m ()
 logError = log SevError
 
 withLogger :: Has (Lift IO) sig m => LogCtx m -> LoggerC m a -> m a
-withLogger act = displayConsoleRegions . runLogger act
+withLogger ctx act = displayConsoleRegions (runLogger ctx act) <* sendIO flushConcurrentOutput
```

Comment:
This is unfortunately required, because `displayConsoleRegions` doesn't guarantee flushed output when the action completes. I was seeing occurrences where stdout output (e.g. `--output` mode in `fossa analyze`) was getting dropped

The library has slightly frustrating UI, because `withConcurrentOutput` _does_ guarantee a flush, but you can't use `withConcurrentOutput` and `displayConsoleRegions` at the same time 😬